docs/examples/follow_me.rst

```diff
@@ -4,39 +4,155 @@
+       
+
+   .. note::
+   
+       The examples uses the ``--connect`` parameter to pass the :ref:`connection string <get_started_connect_string>` into the script. 
+       The command above is used to connect to :ref:`SITL <sitl_setup>` running on the local Linux machine via UDP port 14550.
+
+
+On the terminal you should see (something like):
+
+
+.. code-block:: bash
+
+    dronekit-python/examples/follow_me$ python follow_me.py --connect 127.0.0.1:14550
+    
+    Connecting to vehicle on: 127.0.0.1:14550
+    >>> APM:Copter V3.4-dev (e0810c2e)
+    >>> Frame: QUAD
+    Link timeout, no heartbeat in last 5 seconds
+    Basic pre-arm checks
+    Waiting for GPS...: None
+    ...
+    Waiting for GPS...: None
+    Taking off!
+     Altitude:  0.019999999553
+     ...
+     Altitude:  4.76000022888
+    Reached target altitude
+    Going to: Location:lat=50.616468333,lon=7.131903333,alt=30,is_relative=True
+    ...
+    Going to: Location:lat=50.616468333,lon=7.131903333,alt=30,is_relative=True
+    Going to: Location:lat=50.616468333,lon=7.131903333,alt=30,is_relative=True
+    User has changed flight modes - aborting follow-me
+    Close vehicle object
+    Completed
+
+ 
+To stop follow-me you can change the vehicle mode (on a real flight you can just change the mode switch on your RC transmitter). 
+
+.. note:: 
+
+    The terminal output above was created using simulated GPS data 
+    (which is why the same target location is returned every time). It was exited by changing
+    the mode from GUIDED to STABILIZE using MAVProxy.
+    
+
+    
+How does it work?
+=================
+
+Most of the example should be fairly familiar as it uses the same code as other examples for connecting to the vehicle, 
+:ref:`taking off <taking-off>`, and closing the vehicle object. 
+
+The example-specific code is shown below.  All this does is attempt to get a gps socket and read the location in a two second loop. If it is successful it 
+reports the value and uses :py:func:`Vehicle.commands.goto <dronekit.lib.CommandSequence.goto>` to move to the new position. The loop exits when 
+the mode is changed. 
+
+.. code-block:: python
+
+    import gps
+    import socket
+    
+    ...
+
+    try:
+        # Use the python gps package to access the laptop GPS
+        gpsd = gps.gps(mode=gps.WATCH_ENABLE)
+
+        #Arm and take of to altitude of 5 meters
+        arm_and_takeoff(5)
+
+        while True:
+        
+            if vehicle.mode.name != "GUIDED":
+                print "User has changed flight modes - aborting follow-me"
+                break    
+                
+            # Read the GPS state from the laptop
+            gpsd.next()
+
+            # Once we have a valid location (see gpsd documentation) we can start moving our vehicle around
+            if (gpsd.valid & gps.LATLON_SET) != 0:
+                altitude = 30  # in meters
+                dest = Location(gpsd.fix.latitude, gpsd.fix.longitude, altitude, is_relative=True)
```

Comment:
I believe Location will no longer work on master. This is now `LocationGlobal` i beliece?